examples/hotrod/README.md

```diff
@@ -23,22 +23,27 @@ to view the traces. A tutorial / walkthough is available:
 
 An all-in-one Jaeger backend is packaged as a Docker container with in-memory storage.
 
-```
-docker run -d -p6831:6831/udp -p16686:16686 jaegertracing/all-in-one:latest
+```bash
+docker run -d --name jaeger -p6831:6831/udp -p16686:16686 jaegertracing/all-in-one:latest
 ```
 
 Jaeger UI can be accessed at http://localhost:16686.
 
-### Run HotROD Application
+### Run HotROD from source
 
-```
+```bash
 go get github.com/jaegertracing/jaeger
 cd $GOPATH/src/github.com/jaegertracing/jaeger
 make install
 cd examples/hotrod
 go run ./main.go all
 ```
 
+### Run HotROD from docker
+```bash
+docker run --rm -it --link jaeger -p8080-8083:8080-8083 jaegertracing/example-hotrod:latest /go/bin/hotrod-linux all --jaeger-agent.host-port=jaeger:6831
```

Comment:
can we use syntax with ENV var? I think it will make the command less intimidating.

Reply:
yes, I think there are two variants:

* use viper to configure this
* define env var in dockerfile and pass it to this flag (defaults to localhost) -  only for agent host port as it will be used the most

Reply:
I don't think we need viper in hotrod it will complicate code. I will add env var